### PR TITLE
Sahar/samples ovep escalation

### PR DESCRIPTION
**Pull request by sfatimar**
Inference samples revamp. 
Modification in code style
Updating the help mesages

### CODE REVIEW

File: python/OpenVINO_EP/yolov4_object_detection/README.md [hunk 1]
```diff
@@ -16,13 +16,15 @@ The source code for this sample is available [here](https://github.com/microsoft
...
+3. Download the latest version of the [YOLOv4](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/yolov4) model from here.
 
 ## Install ONNX Runtime for OpenVINO Execution Provider
+Please install the onnxruntime-openvino python package from [here](https://github.com/intel/onnxruntime/releases/)
 
-## Build steps
+## Optional Build steps for ONNX Runtime
 [build instructions](https://onnxruntime.ai/docs/build/eps.html#openvino)
 
 ## Reference Documentation
```
Comment (oliviajain) [inline python/OpenVINO_EP/yolov4_object_detection/README.md:30]: In Requirements: 
1. **Add sample link to use sample video bullet** use any sample video with objects as test input to this sample [Download Sample videos](https://github.com/intel-iot-devkit/sample-videos)
2. 
**Change tinyYOLOv2 to yolov4** Download the yolov4 model from the ONNX Model Zoo

File: python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md [hunk 2]
```diff
@@ -9,13 +9,16 @@ The source code for this sample is available [here](https://github.com/microsoft
...
    This model was adapted from [ONNX Model Zoo](https://github.com/onnx/models).Download the latest version of the [tinyYOLOv2](https://github.com/onnx/models/tree/master/vision/object_detection_segmentation/tiny-yolov2) model from here.
 
 ## Install ONNX Runtime for OpenVINO Execution Provider
+Please install the onnxruntime-openvino python package from [here](https://github.com/intel/onnxruntime/releases/)
 
-## Build steps
+## Optional Build steps for ONNX Runtime
 [build instructions](https://onnxruntime.ai/docs/build/eps.html#openvino)
 
 ## Reference Documentation
```
Comment (oliviajain) [inline python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md:24]: In requirements: 

**Add sample link to use sample video bullet** use any sample video with objects as test input to this sample [Download Sample videos](https://github.com/intel-iot-devkit/sample-videos)

**Comment by oliviajain** [comment]
@sfatimar it seems you may need to merge in master too.

File: python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md (hunk 2 above)
Comment (sfatimar) [reply python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md:24]: Resolved

File: python/OpenVINO_EP/yolov4_object_detection/README.md (hunk 1 above)
Comment (sfatimar) [reply python/OpenVINO_EP/yolov4_object_detection/README.md:30]: Resolved

**Review by oliviajain — APPROVED** [review]

**Pull request merged**